nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+                mountPath: /data/db
+
+    volumeClaimTemplates:
+    - metadata:
+        name: data
+        labels:
+          app: ${APPLICATION_NAME}-storage
+        annotations:
+          volume.beta.kubernetes.io/storage-class: ${STORAGE_CLASS}
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: ${VOLUME_MONGODB_CAPACITY}
+
+
+############################################################################################################
+# Nexus service
+############################################################################################################
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    labels:
+      app: nexus
+    name: nexus
+  spec:
+    ports:
+      - name: 8081-tcp
+        port: 8081
+        protocol: TCP
+        targetPort: 8081
+    selector:
+      app: nexus
+      deploymentconfig: nexus
+    sessionAffinity: None
+    type: ClusterIP
+
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    annotations:
+      openshift.io/host.generated: 'true'
+    labels:
+      app: nexus
+    name: nexus
+  spec:
+    host: nexus-${APPLICATION_NAME}.${DOMAIN_SUFFIX}
+    path: /nexus
+    port:
+      targetPort: 8081-tcp
+    to:
+      kind: Service
+      name: nexus
+      weight: 100
+    wildcardPolicy: None
+
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    labels:
+      app: nexus
+    name: nexus
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+      - annotations:
+          openshift.io/imported-from: sonatype/nexus
+        from:
+          kind: DockerImage
+          name: sonatype/nexus
+        importPolicy: {}
+        name: latest
+        referencePolicy:
+          type: Source
+
+- apiVersion: v1
+  kind: Secret
+  type: kubernetes.io/basic-auth
+  metadata:
+    labels:
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-nexus-credentials
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    username: admin
+    password: ${NEXUS_PASSWORD}
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ${APPLICATION_NAME}-nexus-service
+  data:
+    url: http://nexus-${APPLICATION_NAME}.${DOMAIN_SUFFIX}/nexus/service/local
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      volume.beta.kubernetes.io/storage-class: ${STORAGE_CLASS}
+    name: nexus-pv
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${VOLUME_NEXUS_CAPACITY}
+
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    labels:
+      app: nexus
+    name: nexus
```

Comment:
should contain `${APPLICATION_NAME}` so that we can deploy several templates on the same project

Reply:
Is it only at these 2 areas (Nexus DeploymentConfig `labels > app:nexus` and `name: nexus`)? or should I also change it somewhere else (for instance when there is `deploymentconfig: nexus`) ?

Reply:
done